dictionary.py

```diff
@@ -166,3 +166,131 @@
             
 for name_ in surnames_set:
     print(name_)
+
+#13. Вывести список отделов со средним налогом на сотрудников этого отдела.
+
+for employers_info in departments:
+    department_name = employers_info['title']
+    for tax in taxes:
+        if department_name == "HR department":
```

Comment:
Решение с захардкоденными названиями отделов плохо поддерживает и плохъо масштабируется:
1. при малейшем изменении в названиях отделов тут придётся очень внимательно всё менять руками
2. для добавления третьего отдела добавится ещё одна ветка if
Должно быть что-то вроде:
```suggestion
        if department_name == tax["department"]:
```